tests/http/clients/ws-data.c

```diff
@@ -118,6 +118,13 @@ static CURLcode recv_binary(CURL *curl, char *exp_data, size_t exp_len)
       fprintf(stderr, "EAGAIN, sleep, try again\n");
 #ifdef _WIN32
       Sleep(100);
+#elif defined (__TANDEM)
+      /* NonStop only defines usleep when building for a threading model */
+# if defined (_PUT_MODEL_) || defined(_KLT_MODEL_)
+      usleep(100*1000);
+# else
+      sleep(100);
+# endif
```

Comment:
is there really a point to do it conditionally like this on NonStop? If it has a sleep() that does milliseconds, why not use that always?

Reply:
> is there really a point to do it conditionally like this on NonStop? If it has a sleep() that does milliseconds, why not use that always?

`sleep()` is seconds only and process blocking, so a bad idea when building in PUT or KLT threads. `usleep()` is safer in threads, but not available without threads - I do not know way, but that is what I have to work with.